### DISCUSSION TITLE
Ark clustering

### DISCUSSION THREAD

**Discussion by dbenner1255** [pterodactyl/panel#4800] [Support]
Hello, I am just trying to figure out if there is a walkthrough on setting up a ark cluster. I have been searching. I have no problem launching different ark servers but wanted to be able to cluster them so users can jump to different maps. I have not found a whole lot of info on it for Pterodactyl. Any help or pointers would be great.

We have a dedicated machine with plenty of storage and ram so system resources is not a problem.

Thanks again!

**Comment by parkervcp** [comment]
There is a guide pinned in the custom egg support channel

**Reply by dbenner1255** [reply]
Thanks for answering. However I do not know where this channel is. I have joined discord and looked everywhere on github I can think of. Do you have a link to this guide? Thanks!

**Reply by dbenner1255** [reply]
lol found it. Thanks.
